src/python/pants/backend/cargo_bootstrap/register.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
```

Comment:
This should probably be defined under `pants-plugins`, which would send the signal that this is a backend that is only used in the pantsbuild/pants repo, and not actually shipped (and we shouldn't ship it!).